pkg/cloudprovider/cloudapi/azure/azure_nsg_rules.go

```diff
@@ -458,60 +460,67 @@ func convertToAzureAddressPrefix(ruleIPs []*net.IPNet) (*string, []*string) {
 // convertToInternalRulesByAppliedToSGName converts azure rules to securitygroup.IngressRule and securitygroup.EgressRule and split
 // them by security group names.
 func convertToInternalRulesByAppliedToSGName(azureSecurityRules []*armnetwork.SecurityRule,
```

Comment:
This is going to change to handle user created rules which do not have At group in the Rule.